tests/bagatom/test_updateObjectFromXML.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+
+from copy import deepcopy
```

Comment:
I could be wrong but I believe [PEP-8](https://www.python.org/dev/peps/pep-0008/#imports) requires that the pytest and copy imports be switched around, as copy is from the standard library and pytest is a third-party package. Do you have a reason for the ordering?